.github/workflows/stale.yml

```
@@ -0,0 +1,43 @@
+##############################################################################
+##############################################################################
+#
+# NOTE!
+#
+# Please read the README.md file in this directory that defines what should 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)